proto/umee/incentive/v1/genesis.proto

```diff
@@ -21,59 +21,55 @@ message GenesisState {
   repeated AccountUnbondings account_unbondings  = 10 [(gogoproto.nullable) = false];
 }
 
-// TotalBond tracks the amount of coins of one uToken denomination bonded to a
-// given reward tier by all accounts. Used by queries TotalBonded and TotalUnbonding.
-message TotalBond {
-  uint32                   tier   = 1;
-  cosmos.base.v1beta1.Coin amount = 2 [(gogoproto.nullable) = false];
-}
```

Comment:
Since individual bonds are stored in genesis state, storing their sum is redundant and would require another check in genesis validation (to confirm that total = sum of all individual bonds).

Instead, during init genesis the totalbond in KVStore starts from zero and increases whenever an individual bond is updated. This is a side effect of the `setBonded(ctx sdk.Context, addr sdk.AccAddress, uToken sdk.Coin)` updating total bonded whenever it is called.